js/src/3rdparty/etherscan/util/address.js

```diff
@@ -0,0 +1 @@
+../../../abi/util/address.js
```

Comment:
It is not part of the public interface of the ABI. The API explicitly exposes it and will maintain it. Basically referring it from the ABI is like using `_prop` on an interface - only really meant for internal consumption, it can change.

The API crosses that line to not duplicate code, but this ABI/API interaction still needs to find a clean way of handling this transgression - the reason it was done this way around was to only have dependencies from API -> ABI, not the other way around. (So the API import is wrong and it is a known wrong, but at least only a single wrong... we should not add more.)